sample/Kmd.Logic.Audit.Client.AspnetCoreSample/Kmd.Logic.Audit.Client.AspnetCoreSample.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Do not change sample projects in this PR. We need to bump .NET Core version for sample projects, but let's do that within other PR. Create PBI for this

Reply:
reverted